pandas/tests/io/formats/data/html/gh54167_expected_output.html

```diff
@@ -0,0 +1,10 @@
+<table border="1" class="dataframe">
```

Comment:
Can you insert this as an HTML string in the test instead?

Reply:
@mroeschke yeah sure, let me update!